xds/src/test/java/io/grpc/xds/XdsClientImplTestBase.java

```diff
@@ -3201,10 +3210,8 @@ public void streamClosedAndRetryWithBackoff() {
     call.verifyRequest(RDS, RDS_RESOURCE, "5", "6764", NODE);
 
     call.sendError(Status.DEADLINE_EXCEEDED.asException());
-    verify(ldsResourceWatcher, times(3)).onError(errorCaptor.capture());
-    verifyStatusWithNodeId(errorCaptor.getValue(), Code.DEADLINE_EXCEEDED, "");
-    verify(rdsResourceWatcher, times(3)).onError(errorCaptor.capture());
-    verifyStatusWithNodeId(errorCaptor.getValue(), Code.DEADLINE_EXCEEDED, "");
+    verify(ldsResourceWatcher, times(2)).onError(errorCaptor.capture());
+    verify(rdsResourceWatcher, times(2)).onError(errorCaptor.capture());
```

Comment:
Why are we skipping verifying `errorCaptor.getValue()` for both `ldsResourceWatcher` and `rdsResourceWatcher` ?

Reply:
The spec says that only watchers should only be notified when channel shuts down if the resources haven't been received yet.  Both the lds and rds resource have previously been received, so no longer should be notified (which is why the times changed from 3 to 2) and therefore wouldn't have a status update.

Reply:
This is where we are testing that "When the ads stream is closed only send errors to subscribers that haven't yet gotten results"